examples/flows/hello-world-yaml/definition.yaml

```diff
@@ -0,0 +1,14 @@
+Comment: Run the Hello World Action as the Flow's identity
```

Comment:
NICE - I like that we can start sharing examples for how Flow definitions in yaml look like. I for one think it helps with readability a lot